src/content/deadlines/02_cfp.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Call for Proposals
+subtitle: Opens Until 27 January
```

Comment:
This might sound like it is opening on 27 January, instead of ending.

Reply:
Hmmm doesn't sound like that to me. But to avoid confusion, how about Opens Now till 27 January, or just put the whole date range "Opens 10-27 January"?

Reply:
Sure that works too! I'll get that in just now.